config/manager/manager.yaml

```diff
@@ -1,39 +1,46 @@
-apiVersion: v1
```

Comment:
I'm not sure we ever used, and I'm also not sure we should ever use, the manifests under `config/`

Reply:
This version is auto generated, a later patch will move deploy/operator.yaml to here.
Dir config/ is actually used in order to create the crd/csv.

Reply:
Is the "old" deploy/operator.yaml - so used